gcloud-java-pubsub/baseline/src/main/java/com/google/cloud/pubsub/spi/v1/SubscriberSettings.java

```diff
@@ -408,8 +451,14 @@ public Builder provideChannelWith(ConnectionSettings settings) {
     }
 
     @Override
-    public Builder setExecutor(ScheduledExecutorService executor) {
-      super.setExecutor(executor);
+    public Builder provideChannelWith(Credentials credentials) {
```

Comment:
If would be nice if there was a way of providing no credentials (e.g. for testing with a local emulator) to avoid having to pass a channel with `provideChannelWith`.